clever/package.xml

```diff
@@ -36,6 +37,7 @@
   <depend>rosbridge_server</depend>
   <depend>web_video_server</depend>
   <depend>tf2_web_republisher</depend>
+  <depend>ws281x</depend>
```

Comment:
Not sure this should be in the `clever` package, to be honest. `ws281x` is highly platform-dependent right now, is not in any of the ROS repositories, and will prevent anyone from installing the package dependencies. Maybe create a separate, third package?

Reply:
Third package for what?

Reply:
For platform-dependent, Raspberry-only packages. This will come to bite us as soon as we wish to port this to another platform.

Reply:
So are we removing this hard dependency?